### PR TITLE
chore: update django and djangorestframework-stubs

### PR DESCRIPTION
**Pull request by anonrig**

### CODE REVIEW

File: packages.ini [hunk 1]
```diff
@@ -331,6 +332,7 @@ validate_incorrect_missing_deps = six
 [djangorestframework-stubs==3.14.0]
 [djangorestframework-stubs==3.14.2]
 [djangorestframework-stubs==3.14.4]
+[djangorestframework-stubs==3.14.5]
```
Comment (asottile-sentry) [inline packages.ini:335]: we use a fork of this in sentry -- we'll want to rebase that

**Review by asottile-sentry — COMMENTED** [review]
otherwise looks good!

File: packages.ini (hunk 1 above)
Comment (asottile-sentry) [reply packages.ini:335]: actually I've already rebased on this one!

**Review by asottile-sentry — APPROVED** [review]

File: packages.ini (hunk 1 above)
Comment (anonrig) [reply packages.ini:335]: thank you

**Pull request merged**